generatePoaXml.py

```diff
@@ -227,8 +227,15 @@ def set_article_datasets_header(self, parent):
 
     def set_data_availability(self, parent, poa_article):
         if poa_article.data_availability:
-            p_tag = SubElement(parent, "p")
-            p_tag.text = poa_article.data_availability
+            tag_name = 'p'
```

Comment:
`'p'` & `"p"` are used quite alot in `generatePoaXml`, could possibily move it into a module constant?

Reply:
I'm not too sure if the use of constants is the best thing in this case. I can be convinced to follow a particular style though if you could describe where constants would be preferred in some or all situations. This particular library too, once I integrate the new ones into production, I hope to archive this old one, so it doesn't really need to be cleaned up; there are already repos to replace this repo (not to imply the other repos are any cleaner of course, but we could create the constants in the repos that will be actively developed).